torch/onnx/_internal/exporter.py

```diff
@@ -286,18 +334,18 @@ def dynamo_export(
         ).save("my_model.onnx")
     """
 
-    resolved_export_options = _resolve_export_options(export_options)
+    resolved_export_options = ResolvedExportOptions(export_options)
 
     _assert_dependencies(resolved_export_options)
 
-    from torch.onnx._internal.exporter_impl import DynamoExporter
+    from torch.onnx._internal.exporters import DynamoOptimizeExporter
 
-    return DynamoExporter(
+    return DynamoOptimizeExporter(
```

Comment:
Suggest `DynamoExportExporter` as default

Reply:
Agreed, but in another PR where we combine `DynamoOptimizeExporter` and `DynamoExportExporter` (and it can just be `DynamoExporter` again).